### ISSUE TITLE
Cannot post any new content. Queue function

### ISSUE THREAD

**Issue by thebadkitty** [bdsmlr-development/bdsmlr-updates#26]

**Comment by boisricandy** [comment]
Me too Cant post new content to queue.

**Comment by Someonemustnothavethis** [comment]
This is garbage.  I can't upload anything new.
This is just the latest problem with a site that is sl-o-o-o-o-w as nails with 'features' that don't work.  
Your archive view is a joke and posts get dropped in archive view after a year or so.
Even with the porn ban, Tumblr is still much better, and there's ways to work around the ban.  I may even move to Twitter.  either way, I'm bailing.  with the amount of advertising on the site, you'd think you'd have the money to make it work.

**Comment by notreadyforsen** [comment]
I'm having the same problem, unable to post any new pics. Text posts aren't a problem but I can't insert pics into them. I tried the basics, clear the cache, switched browsers (I use Brave, tried with Firefox), but no apparent change. I'm getting the standard "Sorry, one of the images could not be uploaded, please make sure image is 10 MB or under in size and jpg, gif or png file" error message. My pics are well below the max size, and all are in one of the supported formats. Any help or assistance would be greatly appreciated 🙂

**Comment by thebadkitty** [comment]
Yes, it’s happening again. On multiple browsers.  Is there any point in reporting anything anymore? It’s not being addressed.
